PointOfDispensingApp/src/brokerRequests.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+
+// define functions for API requests here
+export default {
+  // Search patient for patient retrieval
+  searchPatient: (data) => {
```

Comment:
Add comments describing "data" parameter for your new methods. Even if it is a reference to a model somewhere.

Reply:
Done